### PR TITLE
GN3: Imaryll, Elfhame Elite (+ refactor another quasi-keyword -> static)

### PR DESCRIPTION
**Pull request by Northmoc**

### CODE REVIEW

**Review by Agetian — APPROVED** [review]

**Comment by Hanmac** [comment]
i didn't checked all of them, is there one where it isn't optional?

File: forge-ai/src/main/java/forge/ai/AiBlockController.java [hunk 1]
```diff
@@ -204,7 +205,7 @@ private void makeGoodBlocks(final Combat combat) {
                     }
                     blocker = ComputerUtilCard.getWorstCreatureAI(killingBlockers);
                 // 2.Blockers that won't get destroyed
-                } else if (!attacker.hasKeyword("You may have CARDNAME assign its combat damage as though it weren't blocked.")
+                } else if (StaticAbilityAssignCombatDamageAsUnblocked.assignCombatDamageAsUnblocked(attacker)
```
Comment (tool4ever) [inline forge-ai/src/main/java/forge/ai/AiBlockController.java:208]: missing !

**Comment by Northmoc** [comment]
> i didn't checked all of them, is there one where it isn't optional?

Yes, Outmaneuver

Seems to be set up right

File: forge-ai/src/main/java/forge/ai/AiBlockController.java (hunk 1 above)
Comment (Northmoc) [reply forge-ai/src/main/java/forge/ai/AiBlockController.java:208]: Good eye

I left that there to test you... heh

**Review by tool4ever — APPROVED** [review]

**Pull request merged**